var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -382,12 +396,6 @@ def nvptx_install(self):
             make()
             make('install')
 
-    @property
-    def build_targets(self):
-        if sys.platform == 'darwin':
-            return ['bootstrap']
-        return []
```

Comment:
At one time, the Homebrew recipe recommended `make bootstrap` over `make`, but this is no longer the case. Unifying the package so that every OS builds the same.